package.json

```diff
@@ -3,8 +3,11 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
+    "prepare": "husky install",
     "dev": "yarn && nuxi dev",
     "build": "nuxi generate",
+    "lint": "yarn && eslint . --ext .js,.ts,.json",
+    "lint:check": "yarn && eslint . --ext .js,.ts,.json",
```

Comment:
what;s the thought between having lint and lint:check? Could we just use lint?

Reply:
Ahh... they were different when prettier was included but now are the same. If we decide not to add prettier, I will consolidate them

Reply:
ah makes sense. Let's combine for now. We might not actually have to include prettier in the codebase, but rather just a local plugin for your IDE.